PIL/Image.py

```diff
@@ -695,9 +698,9 @@ def __getstate__(self):
             self.tobytes()]
 
     def __setstate__(self, state):
-        # type: (List) -> None
+        # type: (Tuple) -> None
         Image.__init__(self)
-        self.tile = []
+        self.tile = []  # type: List[Tuple] ## FIXME More detail? Should tile be in __init__ & _new?
```

Comment:
I've added some comments with some notes of potential values, but it's not used in the current round of actual type-checking, within Image.py.